test/test.bzl

```diff
@@ -0,0 +1,12 @@
+
+def lin_test(name, tags=[]):
```

Comment:
This is a good idea, having this macro. I'm doing the same thing over on the FSW side to reduce code duplication.

Reply:
One thing worth considering is adding default arguments for srcs, deps, and tags and adding them to the existing values. This way the macro is maximally extensible and applicable to future tests.

Reply:
Yeah this is for sure a good point and something we should do on the flight software side of things! I'm going to leave it for now given this will be, I think, the last infrastructure/big change code wise to this repo in a long time.